learningresources/tests/base.py

```diff
@@ -155,5 +177,5 @@ def get_course_single_tarball(self):
         Returns:
             path (unicode): absolute path to tarball.
         """
-        path = join(abspath(dirname(__file__)), "testdata", "courses")
-        return self.copy_file(join(path, "single.tgz"))
+        path = join(abspath(dirname(__file__)), "testdata", "courses", "toy")
+        return self._make_archive(path, False, '.tgz')
```

Comment:
`.tar.gz` and `.tgz` are interchangeable. There is no difference. I did a quick Google search and found one claim that `.tgz` came from old filename length limits in DOS. 

Either way, it's fine. You can change the extension of a `.tgz` to a `.tar.gz` (or vice versa) and still decompress it with the same command and get the same files out.